apps/frontend/src/pages/overview/overview.tsx

```diff
@@ -0,0 +1,13 @@
+import { useAppSelector } from "~/libs/hooks/hooks.js";
+
+import { WelcomeHeader } from "./components/welcome-section-header/welcome-section-header.js";
+
+const Overview: React.FC = () => {
+	const { user } = useAppSelector((state) => state.auth);
+
+	return user ? (
+		<WelcomeHeader firstName={user.firstName} lastName={user.lastName} />
+	) : null;
```

Comment:
```suggestion
		<WelcomeHeader user={user} />
```
we should always have user on this page

Reply:
Hi, in the slice we have state user which is of type UserAuthResponseDto | null, so even if user will be authorized after the  page overview will select this state, the overview page wont know that the user already exist it'll obtain UserAuthResponseDto | null, Oleg give me advice to use non-null-assertion but it cause eslint error:
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-trackmates/assets/126669822/b31e0872-e574-4eb8-9698-a14765599f0a)
So can you give me advice how to be with that?

Reply:
Or i should simply pass empty strings to the firstName and lastName if user is null?